Description: Update package metadata for RISC-V64
 Changes maintainer and restricts architecture to riscv64.
 Updates Vcs fields to point to our fork.
Author: Bruno Verachten <gounthar@gmail.com>
Forwarded: not-needed
Last-Update: 2025-10-18

--- a/debian/control
+++ b/debian/control
@@ -2,7 +2,8 @@
 Section: admin
 Priority: optional
 Standards-Version: 4.7.0
-Maintainer: Debian Go Packaging Team <team+pkg-go@tracker.debian.org>
+Maintainer: Bruno Verachten <gounthar@gmail.com>
+XSBC-Original-Maintainer: Debian Go Packaging Team <team+pkg-go@tracker.debian.org>
 Uploaders: Arnaud Rebillout <arnaudr@kali.org>
           ,Dmitry Smirnov <onlyjob@debian.org>
           ,Tim Potter <tpot@hpe.com>
@@ -147,8 +148,8 @@
     ,git
     ,golang-github-adalogics-go-fuzz-headers-dev
 Homepage: https://mobyproject.org
-Vcs-Browser: https://salsa.debian.org/go-team/packages/docker
-Vcs-Git: https://salsa.debian.org/go-team/packages/docker.git
+Vcs-Browser: https://github.com/gounthar/docker-for-riscv64
+Vcs-Git: https://github.com/gounthar/docker-for-riscv64.git
 Rules-Requires-Root: no
 XS-Go-Import-Path: github.com/docker/docker
     ,github.com/docker/cli
@@ -156,7 +157,7 @@
     ,github.com/moby/ipvs
 
 Package: docker.io
-Architecture: linux-any
+Architecture: riscv64
 Pre-Depends: ${misc:Pre-Depends}
 Depends: ${misc:Depends} ,${shlibs:Depends}
     ,adduser
